buildenv/docker/riscv64/debian11/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# Copyright IBM Corp. and others 2022
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+
+# Create the OMR build environment with Debian 11.7.
+
+FROM debian:11.7-slim AS base
+
+# Workaround for a hang during docker build.
+ENV TZ=America/Toronto
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update && apt-get install -y \
+	binfmt-support \
+        bison \
+	build-essential \
+        ccache \
+        cmake \
+        flex \
+	gcc-riscv64-linux-gnu \
+        gdb \
+        git \
+	g++-riscv64-linux-gnu \
+        libelf-dev \
+        libdwarf-dev \
+	libglib2.0-dev \
+	lsb-release \
+	mmdebstrap \
+        ninja-build \
+	pkg-config \
+        python3 \
+	qemu-user-static \
+	time \
+	tzdata \
+        vim \
+	wget \
+        && rm -rf /var/lib/apt/lists/*
+
+# Add user home and copy authorized_keys and known_hosts.
+RUN groupadd -r jenkins \
+&& useradd -rm -u 1000 -g jenkins jenkins \
+&& mkdir -p /home/jenkins/.ssh \
+&& echo "Host git.eclipse.org*\n\tStrictHostKeyChecking no\n" > /home/jenkins/.ssh/config \
```

Comment:
Do we have anything on git.eclipse.org used in these builds?

Reply:
I don't recall, but likely a c&p from another dockerfile and just not enough clean up so I'll remove.